tests/unit/test_xtce/test_comparisons.py

```diff
@@ -4,172 +4,171 @@
 
 from space_packet_parser import common
 from space_packet_parser.exceptions import ComparisonError
-from space_packet_parser.xtce import comparisons, XTCE_NSMAP
+from space_packet_parser.xtce import XTCE_1_2_XMLNS, comparisons
 
 
 @pytest.mark.parametrize(
     ('xml_string', 'test_parsed_data', 'current_parsed_value', 'expected_comparison_result'),
     [
-        ("""
-<xtce:Comparison xmlns:xtce="http://www.omg.org/space/xtce"
+        (f"""
+<xtce:Comparison xmlns:xtce="{XTCE_1_2_XMLNS}"
```

Comment:
Do you need the namespace defined here, or can we just remove it altogether? Either `<Comparison>` or `<xtce:Comparison>` without the trailing text.

Reply:
The tests use a standard fixture now that generates an `xtce_parser` that is namespace aware, but since it's using the same fixture for all tests, all tests need to use the same namespacing. We could remove it from _all_ the tests that use the `xtce_parser` fixture...